src/sas/sasgui/perspectives/fitting/fitting.py

```diff
@@ -1732,6 +1732,12 @@ def _complete1D(self, x, y, page_id, elapsed, index, model,
             @param unsmeared_error: data error, rescaled to unsmeared model
         """
         try:
+            number_finite = np.count_nonzero(np.isfinite(y))
+            if number_finite == 0:
+                logger.error("Using the present parameters the model does not return any finite value. ")
+                wx.PostEvent(self.parent, StatusEvent(status=\
```

Comment:
please don't use line continuation. You are using parentheses, you shouldn't need to use line continuation.